awscli/examples/rum/put-rum-events.rst

```diff
@@ -0,0 +1,12 @@
+**To send telemetry events about your application performance and user behavior to CloudWatch RUM**
+
+The following ``put-rum-events`` sends telemetry events about your application performance and user behavior to CloudWatch RUM. If the command succeeds, no output is returned. ::
+
+	aws rum put-rum-events \
+		--app-monitor-details "{\"id\":\"r6fj89df-5rt3-55h6-9875-bg6c7uud94fg\",\"name\":\"MyWebApp\",\"version\":\"1.0.0\"}" \
```

Comment:
Command examples need to follow standard unix-like quoting rules, where strings are surrounded by single quotes. With single quotes you no longer need to escape your double quotes, so it'll look similar to the following:

```suggestion
		--app-monitor-details '{"id":"r6fj89df-5rt3-55h6-9875-bg6c7uud94fg","name":"MyWebApp","version":"1.0.0"}' \
```

Be sure to fix all your command options following these quoting rules.